src/pcre2_jit_compile.c

```diff
@@ -2362,7 +2362,7 @@ SLJIT_ASSERT(stackpos == STACK(stacktop));
 typedef struct delayed_mem_copy_status {
   struct sljit_compiler *compiler;
   int store_bases[RECURSE_TMP_REG_COUNT];
-  int store_offsets[RECURSE_TMP_REG_COUNT];
+  sljit_sw store_offsets[RECURSE_TMP_REG_COUNT];
```

Comment:
These are intentionally small values.

Reply:
Without this change, `delayed_mem_copy_move`'s assignment into `store_offsets` can wind up truncated.  Attempting to narrow the type of its `store_offset` argument instead reveals that three call sites pass in 64-bit values.